src/components/Book.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const Book = (props) => {
+  const dispatch = useDispatch();
+  const {
+    book,
+    author,
+    percent,
+    chapter,
+  } = props;
+  const {
+    id,
+    title,
+    category,
+  } = book;
+
+  return (
+    <li className="book-list" key={id}>
+      <div>
+        <small><b className="category">{category}</b></small>
+        <h6 className="title">{title}</h6>
+        <small className="author">{author}</small>
+        <div>
+          <button type="button" className="list-btn">Comments</button>
+          <button type="button" className="list-btn" onClick={() => dispatch(removeBook(id))}>Remove</button>
+          <button type="button" className="list-btn">Edit</button>
+        </div>
+      </div>
+      <div>
+        <span className="circle" />
+        <div>
+          <h4 className="percentage">
+            {percent}
+            %
+          </h4>
+          <small>Completed</small>
+        </div>
+      </div>
+      <div>
+        <p>CURRENT CHAPTER</p>
+        <p>{chapter}</p>
+        <button type="button">UPDATE PROGRESS</button>
+      </div>
+    </li>
+  );
+};
+
+Book.defaultProps = {
+  author: 'Unknown',
+  percent: 0,
+  chapter: 'Introduction',
+};
+
+Book.propTypes = {
+  book: PropTypes.shape({
+    id: PropTypes.number.isRequired,
```

Comment:
- [ ] Kindly resolve the warning in the browser console as it helps write more conventional code. When creating a book in the **AddBook** component this line `const id = uuidv4();` generates a `string` type whereas here you are validating for `number`. Kindly fix this.

  ![reddy-log](https://user-images.githubusercontent.com/36542986/155116782-f2427223-be2d-4f2c-970e-6e62aa778550.jpg)